lib/index.js

```diff
@@ -25,13 +25,14 @@ class Service extends AdapterService {
           defaultParams: {
             echoParams: 'none'
             // debug:"all"
-          }
+          },
+          escapeFn: options.escapeFn || function (key, value) { return {key, value} }
         },
         options,
```

Comment:
The logic `options.escapeFn || function (key, value) { return {key, value} }` isn't really necessary because of the assigning of options. It would already be overwritten by this.

Even I think it would be more readable using spread operators instead of Object.assign.